ui/src/types/query.tsx

```diff
@@ -8,7 +8,7 @@ export interface Query {
   groupBy: GroupBy
   areTagsAccepted: boolean
   rawText: string | null
-  range?: TimeRange | null
+  range?: DurationRange | null
   source?: string
```

Comment:
Why switch this from `TimeRange`?

Reply:
switched because the backend calls this struct `DurationRange`, so i decided to have the client match the backend in terms of naming